conda/plugins/manager.py

```diff
@@ -366,18 +360,33 @@ def get_virtual_packages(self) -> tuple[CondaVirtualPackage, ...]:
     def get_reporter_backends(self) -> tuple[CondaReporterBackend, ...]:
         return tuple(self.get_hook_results("reporter_backends"))
 
-    def get_reporter_backend(self, name: str) -> CondaReporterBackend | None:
-        for reporter_backend in self.get_reporter_backends():
+    def get_reporter_backend(self, name: str) -> CondaReporterBackend:
```

Comment:
Let's improve this with a more strict fallback handling, similar to the solver loading behavior:

```python
    def get_reporter_backend(self, name: str) -> CondaReporterBackend:
        """
        Attempts to find a reporter backend while providing a fallback option if it is
        not found.

        This method must return a valid ``CondaReporterBackend`` object.
        """
        reporter_backends = {
            reporter_backend.name: reporter_backend
            for reporter_backend in self.get_reporter_backends()
        }
        reporter_backend = reporter_backends.get(name, None)
        if reporter_backend is None:
            log.warning(
                f'Unable to find reporter backend: "{name}"; '
                f'falling back to using "{DEFAULT_CONSOLE_REPORTER_BACKEND}"'
            )
            return reporter_backends.get(DEFAULT_CONSOLE_REPORTER_BACKEND)
        else:
            return reporter_backend

```